couchbase/operations/document_lookup_in.cxx

```diff
@@ -62,6 +62,8 @@ lookup_in_request::make_response(error_context::key_value&& ctx, const encoded_r
         for (size_t i = 0; i < encoded.body().fields().size(); ++i) {
             const auto& res_entry = encoded.body().fields()[i];
             response.fields[i].status = res_entry.status;
+            response.fields[i].ec =
```

Comment:
This is causing build failures on mac because mcbp_session requires openssl. Maybe map_status_code could be moved to a separate namespace